client.py

```diff
@@ -119,7 +119,7 @@ def handle_connection(self, server):
                 response_info = [chosen_category]
 
             elif parsed_message.code == MessageType.SPIN:
-                [player_id] = parsed_message.args  # TODO: Why is this client receiving its own player ID?
+                _ = parsed_message.args
```

Comment:
I think we should have the server send the `player_id` of the player taking the turn, that way the client can check if that is their ID and decide whether to display the "spin" button/prompt.